packages/api-key-service/src/functions/paginate.ts

```diff
@@ -13,6 +13,9 @@ type Pagintation<T> = {
 }
 
 export const paginateApiKeys = httpHandler<void, Pagintation<ApiKeyModel>>({
+  defaultOutputHeaders: {
```

Comment:
Should the default headers live in a constant?

Reply:
yea why not!